src/nunavut/lang/cpp/templates/_fields_as_variant.j2

```diff
@@ -3,54 +3,49 @@
  # Copyright (C) 2021  UAVCAN Development Team  <uavcan.org>
  # This software is distributed under the terms of the MIT License.
 #}
-    using {{ composite_type.short_name.capitalize() | id }}VariantType = std::variant<
+    using VariantType = class VariantType_ final : public std::variant<
```

Comment:
What is the purpose of this alias? Why not simply `class VariantType final : public std::variant<>`?

Reply:
Yeah, it's pointless. Will change